modules/cnv-edit-boot-order-web.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-boot-order.adoc
+
+[id="cnv-edit-boot-order-web_{context}"]
+= Editing a boot order list in the web console
+
+Edit the boot order list in the web console.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+
+. Select a virtual machine to open the *Virtual Machine Overview* screen.
+
+. Click the pencil icon that is located on the right side of *Boot Order*.
+
+. Choose the appropriate method to move the item in the boot order list:
+
+* If you do not use a screen reader, hover over the arrow icon next to the item that you want to move, drag the item up or down, and drop it in a location of your choice.
+
+* If you use a screen reader, tap the *Up* or *Down* key to move the item in the boot order list. Then, tap the *Tab* key to drop the item in a location of your choice.
```

Comment:
Are you referring to keyboard keys here? A few points, if so: 

- These do not need to be bolded. 
- For arrow keys, call them the `Up Arrow key`, and `Down Arrow key`. 
- The ISG also says to use "press" for keys that don't result in output on the screen (pg 218-219). So please s/tap/press :)